### PR TITLE
Fix numeric noise in target computation

### PR DESCRIPTION
**Pull request by Vayel**
See #126

### CODE REVIEW

**Comment by Vayel** [comment]
@MaxHalford is it ok for you? :)

File: ethik/query.py [hunk 1]
```diff
@@ -146,7 +146,10 @@ def from_taus(
 
         q_mins = quantiles.loc[q[0]].to_dict()
         q_maxs = quantiles.loc[q[1]].to_dict()
-        means = X_test.mean().to_dict()
+        # Cannot use `means = X_test.mean().to_dict()` as it may give a different
+        # result depending on whether the feature is alone in the dataset or not
+        #  Instead, we compute the means for each pandas series independently
+        means = {feature: X_test[feature].mean() for feature in X_test.columns}
```
Comment (MaxHalford) [inline ethik/query.py]: Do you have an example of this?

Comment (Vayel) [reply ethik/query.py]: I wrote a test. Otherwise, in #126  :)

Comment (Vayel) [reply ethik/query.py]: Do you mean I should put it in the comment?

Comment (MaxHalford) [reply ethik/query.py]: No not at all, it's just for my general knowledge! I saw your issue, I just haven't had time to look at it. We're good to go!

**Pull request merged**